packages/performance/src/services/transport_service.ts

```diff
@@ -61,6 +65,11 @@ export function setupTransportService(): void {
   }
 }
 
+export function resetTransportService(): void {
```

Comment:
If this method is used only for testing, should we qualify that with some comments here?

Reply:
Make sense! Added function comments.